diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2017-03-24 21:31:17.000000000 +0100
+++ ./CMakeLists.txt	2017-04-29 02:36:13.904422888 +0200
@@ -68,7 +68,7 @@ list(APPEND ARGUSTV_SOURCES ${ARGUSTV_HE
 
 set(DEPLIBS ${p8-platform_LIBRARIES} tsreader)
 
-find_package(JsonCpp REQUIRED)
+find_package(jsoncpp REQUIRED)
 list(APPEND DEPLIBS ${JSONCPP_LIBRARIES})
 list(APPEND INCLUDES ${JSONCPP_INCLUDE_DIRS})
 
